e2e/specs/st_expander.spec.js

```diff
@@ -42,7 +42,7 @@ describe("st.expander", () => {
     // See https://github.com/streamlit/streamlit/issues/2437
     cy.get(".stButton button").focus();
     cy.get(".main").matchThemedSnapshots("expanders-in-main");
-    cy.get("[data-testid='stSidebar']").matchThemedSnapshots(
+    cy.get("[data-testid='stSidebarUserContent']").matchThemedSnapshots(
```

Comment:
Changing snapshot to the Sidebar's UserContent section - Cypress on GHA doesn't handle the dynamic sidebar height sizing well for this particular test